modules/es-extensions/publisher/asset/restservice/asset.js

```diff
@@ -81,6 +81,25 @@ asset.manager = function(ctx) {
 
             return item;
         },
+        create: function(options) {
+            var name = options.attributes.overview_name;
+            var version = options.attributes.overview_version;
+            var rxt = require('rxt');
+            var am = rxt.asset.createUserAssetManager(ctx.session, this.type);
+            var query = {};
+            query.overview_name = name;
+            var assets = am.search(query);
```

Comment:
Could we change this to :  

var assets = am.getAssetGroup(ctx.assetType);
var isVersionExist = assets.filter(function(asset) {
   return asset.version == version;
});
if(isVersionExist){
     throw <Error>
}
return this._super.create.call(this,options);